support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -342,6 +342,8 @@ export function ThreeTierLanding(): JSX.Element {
 		),
 	);
 
+	const useGenericCheckout =
+		abParticipations.contributionGenericCheckout === 'variant';
```

Comment:
Looking at this - I wonder if we just have the test as `useGenericCheckout` and then start adding more and more routes to it?


```suggestion
	const useGenericCheckout =
		abParticipations.useGenericCheckout === 'variant';
```

Reply:
Ok renamed as suggested (and in abtest - hence done manually)